.changeset/wicked-dancers-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Added a new `center` style mixin to center content horizontally and vertically.
```

Comment:
This looks like a rebase or merge gone wrong 👀

Reply:
sure is, trying again!

Reply:
Fixed!:)

Reply:
But I lost my changeset:'(

Reply:
There, I put it back.